src/analysis/PluginBase.py

```
@@ -214,7 +219,8 @@ def worker_processing_with_timeout(self, worker_id, next_task: FileObject):
         if self.timeout_happened(process):
             result_fo = self._handle_failed_analysis(next_task, process, worker_id, 'Timeout')
         elif process.exception:
-            result_fo = self._handle_failed_analysis(next_task, process, worker_id, 'Exception')
+            _, trace = process.exception
+            result_fo = self._handle_failed_analysis(next_task, process, worker_id, 'Exception', trace=trace)
```

Why ignore the exception and use "Exception" in the line after?